src/less/index.less

```diff
@@ -1,31 +1,5 @@
 // Clever Styles
-// Version 0.6.2
+// Version 0.17.3
```

Comment:
Not necessarily relevant to this PR, though is there a way we can tie the package version to this, or should we not put this here? since I'm pretty sure we won't remember to bump this version frequently